src/chem/dep/dep_wet_ls_mod.F90

```diff
@@ -169,7 +184,19 @@ subroutine wetdep_ls(dt,var,rain,moist,rho,var_rmv,           &
         do k=1,kte
            dvar=max(0.,(moist(i,k,j,p_qc)+moist(i,k,j,p_qi)))
            var_sum_clw(i,j)=var_sum_clw(i,j)+dvar
+           var_sum(i,j)=var_sum(i,j)+var(i,k,j,nv)*rho(i,k,j) !lzhang
```

Comment:
Delete the !lzhang or add more detailed description.

Reply:
agree